src/composables/Git.js

```diff
@@ -189,6 +189,7 @@ export async function createBranchFrom(projectId, newBranchName, branchName) {
  * Add untracked, unstaged or modified files.
  * @param {string} projectId - Id of project.
  * @param {string} filepath - Path of the file to add.
+ * NOTE: Filepath must exclude the root folder.
```

Comment:
@param {string} filepath - Path of the file to add. Filepath must exclude the root folder.

Same  for other

Reply:
done